tests/e2e/core-tests/specs/merchant/wp-admin-settings-shipping-classes.test.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable jest/no-export*/
+
+/**
+ * Internal dependencies
+ */
+const { merchant } = require('@woocommerce/e2e-utils');
+const { lorem, helpers } = require("faker");
```

Comment:
The [package](https://github.com/woocommerce/woocommerce/blob/trunk/tests/e2e/core-tests/package.json) needs to have `faker` in its dependencies. It is working here because `faker` is installed through a dependency in another package.

Reply:
Ah yeah, thanks for spotting it!
Fixed in https://github.com/woocommerce/woocommerce/pull/29467/commits/68df2796a92b971c69c01718ba74ca423036ea52

Reply:
@rodel-calasagsag You can use `npm list {package-name}` to determine which version(s) of a package is/are being used:

```
$ npm list faker
woocommerce@5.3.0 /Users/ronrennick/Sites/solaris/woocommerce
├─┬ @woocommerce/e2e-core-tests@0.1.3 -> /Users/ronrennick/Sites/solaris/woocommerce/tests/e2e/core-tests
│ └── faker@5.5.2 
└─┬ @woocommerce/e2e-utils@0.1.4 -> /Users/ronrennick/Sites/solaris/woocommerce/tests/e2e/utils
  └── faker@5.1.0 
```

We should use the same version of the package everywhere when possible.

Reply:
Thanks @rrennick it's done. I updated the version number to be consistent with the other packages.